apps/meteor/client/lib/utils/getPlanName.ts

```diff
@@ -0,0 +1,34 @@
+export enum PlanName {
+	COMMUNITY = 'Community',
+	STARTER = 'Starter',
+	PRO = 'Pro',
+	PRO_TRIAL = 'Pro_trial',
+	ENTERPRISE = 'Enterprise',
+	ENTERPRISE_TRIAL = 'Enterprise_trial',
+}
```

Comment:
we agreed not to use hardcoded names for licenses, this will come from the license, this is a headache when we decide to change

Reply:
Working on a new version without that, thanks!